### ISSUE TITLE
10.9.10No notification received

### ISSUE THREAD

**Issue by jiemo9527** [RomainPierre7/jellyfin-plugin-TelegramNotifier#8]
server and plugin version：10.9.10
“The previous plugin version was working fine.”

logs:
```
[2024-08-29 14:52:25.532 +08:00] [INF] [15] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2023/2023动漫"
[2024-08-29 14:52:32.086 +08:00] [INF] [17] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2023/2023电影"
[2024-08-29 14:52:37.776 +08:00] [INF] [8] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2022/2022电视剧"
[2024-08-29 14:52:41.466 +08:00] [INF] [20] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2024/2024电影"
[2024-08-29 14:52:44.134 +08:00] [INF] [19] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2024/2024动漫"
[2024-08-29 14:52:50.321 +08:00] [INF] [8] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "46.3.102.124" request
[2024-08-29 14:53:02.902 +08:00] [INF] [18] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2023/2023电视剧"
[2024-08-29 14:53:16.107 +08:00] [INF] [11] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory "/media/2024/2024电视剧"
[2024-08-29 14:53:20.438 +08:00] [INF] [29] Emby.Server.Implementations.ScheduledTasks.TaskManager: "TelegramNotifier Item Added Notifier" Completed after 0 minute(s) and 0 seconds
[2024-08-29 14:54:20.441 +08:00] [INF] [11] Emby.Server.Implementations.ScheduledTasks.TaskManager: "TelegramNotifier Item Added Notifier" Completed after 0 minute(s) and 0 seconds
[2024-08-29 14:55:20.441 +08:00] [INF] [11] Emby.Server.Implementations.ScheduledTasks.TaskManager: "TelegramNotifier Item Added Notifier" Completed after 0 minute(s) and 0 seconds
[2024-08-29 14:55:38.629 +08:00] [INF] [31] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-08-29 14:56:20.445 +08:00] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: "TelegramNotifier Item Added Notifier" Completed after 0 minute(s) and 0 seconds
[2024-08-29 14:56:44.145 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2024/2024动漫"
[2024-08-29 14:56:44.147 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2022/2022电影"
[2024-08-29 14:56:44.147 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2023/2023电影"
[2024-08-29 14:56:44.147 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2022/2022动漫"
[2024-08-29 14:56:44.147 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2024/2024电影"
[2024-08-29 14:56:44.147 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2024/2024电视剧"
[2024-08-29 14:56:44.148 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2022/2022电视剧"
[2024-08-29 14:56:44.148 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2023/2023动漫"
[2024-08-29 14:56:44.148 +08:00] [INF] [29] Emby.Server.Implementations.IO.LibraryMonitor: Stopping directory watching for path "/media/2023/2023电视剧"
[2024-08-29 14:56:44.151 +08:00] [INF] [29] Emby.Server.Implementations.Library.LibraryManager: Validating media library
[2024-08-29 14:56:46.820 +08:00] [WRN] [29] MediaBrowser.Controller.Entities.BaseItem: Library folder "/config/data/playlists" is inaccessible or empty, skipping
[2024-08-29 14:56:47.059 +08:00] [WRN] [31] MediaBrowser.Controller.Entities.BaseItem: Library folder "/config/data/playlists" is inaccessible or empty, skipping
[2024-08-29 14:57:06.764 +08:00] [INF] [34] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP05.mp4\" -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
[2024-08-29 14:57:07.267 +08:00] [INF] [3] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/权力的游戏 第四季 Game of Thrones Season 4 (2014)/Game.of.Thrones.S04E01.Two.Swords.2160p.BluRay.x265.10bit.DV.HDR.TrueHD.7.1.Atmos.mkv\" -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
[2024-08-29 14:57:07.410 +08:00] [INF] [34] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP05.chi.srt\" -threads 0 -v warning -print_format json -show_streams -show_format"
[2024-08-29 14:57:07.521 +08:00] [ERR] [18] MediaBrowser.Providers.MediaInfo.SubtitleResolver: Error getting external streams from "/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP05.chi.srt"
MediaBrowser.Common.FfmpegException: ffprobe failed - streams and format are both null.
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.Providers.MediaInfo.MediaInfoResolver.GetExternalStreamsAsync(Video video, Int32 startIndex, IDirectoryService directoryService, Boolean clearCache, CancellationToken cancellationToken)
[2024-08-29 14:57:08.133 +08:00] [INF] [18] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP06.mp4\" -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
[2024-08-29 14:57:08.486 +08:00] [INF] [34] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP06.chi.srt\" -threads 0 -v warning -print_format json -show_streams -show_format"
[2024-08-29 14:57:08.653 +08:00] [ERR] [29] MediaBrowser.Providers.MediaInfo.SubtitleResolver: Error getting external streams from "/media/2023/2023电视剧/重启人生 ブラッシュアップライフ/重启人生EP06.chi.srt"
MediaBrowser.Common.FfmpegException: ffprobe failed - streams and format are both null.
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.Providers.MediaInfo.MediaInfoResolver.GetExternalStreamsAsync(Video video, Int32 startIndex, IDirectoryService directoryService, Boolean clearCache, CancellationToken cancellationToken)
[2024-08-29 14:57:11.530 +08:00] [INF] [11] Jellyfin.Plugin.MetaShark.Providers.SeriesProvider: [MetaShark] GetSeriesMetadata of [name]: 法证先锋6 [fileName]: 法证先锋6 IsAutomated: True
[2024-08-29 14:57:12.261 +08:00] [INF] [11] Jellyfin.Plugin.MetaShark.Providers.SeriesProvider: [MetaShark] GuessByDouban of [name]: 法证先锋6 [file_name]: 法证先锋6 [year]:  [search name]: 法证先锋6
[2024-08-29 14:57:15.296 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesProvider: [MetaShark] Found douban [id] by first match: 法证先锋6之幸存者的救赎(36534002)
[2024-08-29 14:57:15.298 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesProvider: [MetaShark] GetSeriesMetadata of douban [sid]: 36534002
[2024-08-29 14:57:20.450 +08:00] [INF] [30] Emby.Server.Implementations.ScheduledTasks.TaskManager: "TelegramNotifier Item Added Notifier" Completed after 0 minute(s) and 0 seconds
[2024-08-29 14:57:20.623 +08:00] [INF] [36] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i file:\"/media/2023/2023电视剧/权力的游戏 第四季 Game of Thrones Season 4 (2014)/Game.of.Thrones.S04E01.Two.Swords.2160p.BluRay.x265.10bit.DV.HDR.TrueHD.7.1.Atmos.ass\" -threads 0 -v warning -print_format json -show_streams -show_format"
[2024-08-29 14:57:20.675 +08:00] [ERR] [36] MediaBrowser.Providers.MediaInfo.SubtitleResolver: Error getting external streams from "/media/2023/2023电视剧/权力的游戏 第四季 Game of Thrones Season 4 (2014)/Game.of.Thrones.S04E01.Two.Swords.2160p.BluRay.x265.10bit.DV.HDR.TrueHD.7.1.Atmos.ass"
MediaBrowser.Common.FfmpegException: ffprobe failed - streams and format are both null.
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.MediaEncoding.Encoder.MediaEncoder.GetMediaInfoInternal(String inputPath, String primaryPath, MediaProtocol protocol, Boolean extractChapters, String probeSizeArgument, Boolean isAudio, Nullable`1 videoType, CancellationToken cancellationToken)
   at MediaBrowser.Providers.MediaInfo.MediaInfoResolver.GetExternalStreamsAsync(Video video, Int32 startIndex, IDirectoryService directoryService, Boolean clearCache, CancellationToken cancellationToken)
[2024-08-29 14:57:21.496 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesProvider: [MetaShark] Found tmdb [id]: 249521 by imdb id: tt33078301
[2024-08-29 14:57:22.043 +08:00] [INF] [36] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetImages for item: 法证先锋6之幸存者的救赎 [metaSource]: Douban
[2024-08-29 14:57:22.418 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetBackdrop from tmdb id: "249521"
[2024-08-29 14:57:22.425 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetLogos from tmdb id: "249521" lang: "zh"
[2024-08-29 14:57:22.447 +08:00] [INF] [28] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetImageResponse url: "https://img2.doubanio.com/view/photo/m/public/p2911453621.jpg"
[2024-08-29 14:57:24.563 +08:00] [INF] [11] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetImageResponse url: "http://image.tmdb.org/t/p/original/5ot6Da6gGuwdBnkKQj8ff3R3C4s.png"
[2024-08-29 14:57:24.843 +08:00] [INF] [11] Jellyfin.Plugin.MetaShark.Providers.SeriesImageProvider: [MetaShark] GetImageResponse url: "http://image.tmdb.org/t/p/original/vFGcpnh2GdFJuOAIfVRRqerKdhb.jpg"
```
It seems normal, and new video content has been identified.

plugin settings:
![Snipaste_2024-08-29_15-01-26](https://github.com/user-attachments/assets/a53b44cf-af3a-4dd6-a3cc-70c78b9c5283)
but test message is ok！‘-100......909’ is the telegram group id, and notifications could be received normally before
![2](https://github.com/user-attachments/assets/ab756af5-faf3-40f9-97d7-f0c77c66b163)

**Comment by jiemo9527** [comment]
Help

**Comment by jiemo9527** [comment]
l'm trying

> Hi @jiemo9527, I've just published a fix ([10.9.10.1](https://github.com/RomainPierre7/jellyfin-plugin-TelegramNotifier/releases/tag/10.9.10.1)) for the item added notifier. Update the plugin and let me know if this solves your case or not. Make sure Thread ID is empty if you don't use this feature (no invisible space for example). Enable the session start notifier and reload the jellyfin web page. Are you receiving anything?

This is working fine！

**Comment by RomainPierre7** [comment]
Nice ;) Have a great day !
